src/commands/index.ts

```diff
@@ -21,6 +22,7 @@ export const cmd = subcommands({
     group: commandGroup,
     flatten: commandCreateManifest,
     'lds-fetch-layer': commandLdsFetch,
+    'bm-create-pr': basemapsCreatePullRequest,
```

Comment:
I think we should maybe make a seperate command or groupp it into a basemaps sub command?

Reply:
Fixed